doc/user/content/releases/v0.88.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: "Materialize v0.88"
 date: 2024-02-21
-released: false
+released: true
+patch: 2
```

Comment:
```suggestion
patch: 1
```

Reply:
Ah, just saw that it was cut but won't get rolled out. Thank you!